include/fem-dem/gls_sharp_navier_stokes.h

```diff
@@ -524,13 +498,20 @@ Return a bool that describes  if a cell contains a specific point
 
   PVDHandler ib_particles_pvdhandler;
 
+
   const bool                   SUPG        = true;
   const bool                   PSPG        = true;
   const double                 GLS_u_scale = 1;
```

Comment:
Are these member variables still relevant? Because of the assemblers I don't tthink they are.